Document Sendspin manual IP addresses setting (#654)
music-assistant/music-assistant.io: src/content/docs/player-support/sendspin.md

Standard [player settings](/settings/player-provider/) apply. Specific settings available for this player type are:

> - <b>Manual IP addresses for discovery.</b> Normally Sendspin clients are automatically discovered via mDNS. If automatic discovery does not work in your network setup, you can manually add the IP address or hostname of a Sendspin client. Accepted values are a plain IP address or hostname, optionally with a port, such as `192.168.1.50`, `speaker.local`, or `speaker.local:8928`. For clients using a non-default path or protocol, enter the full WebSocket URL, such as `ws://speaker.local:8928/sendspin` or `wss://speaker.example.com/sendspin`. If only an IP address or hostname is entered, Music Assistant assumes the default Sendspin client endpoint `ws://<host>:8928/sendspin`. Music Assistant will connect to the configured clients directly and keep retrying if a client is offline when MA starts.

Reviewer: ```suggestion
- <b>Manual IP addresses for discovery.</b> Normally Sendspin clients are automatically discovered via mDNS. If automatic discovery does not work due to the network setup, the IP address or hostname of a Sendspin client can be manually entered. Accepted values are a plain IP address or hostname, optionally with a port, such as `192.168.1.50`, `speaker.local`, or `speaker.local:8928`. For clients using a non-default path or protocol, enter the full WebSocket URL, such as `ws://speaker.local:8928/sendspin` or `wss://speaker.example.com/sendspin`. If only an IP address or hostname is entered, Music Assistant assumes the default Sendspin client endpoint `ws://<host>:8928/sendspin`. Music Assistant will connect to the configured clients directly and keep retrying if a client is offline when MA starts.
```

Keeping the language passive third person
